pydbtools/utils.py

```diff
@@ -48,22 +65,21 @@ def check_temp_query(sql: str):
         )
 
 
-def clean_query(sql: str, fmt_opts: dict = None) -> str:
+def clean_query(sql: str, fmt_opts: dict = {}) -> str:
```

Comment:
This needs to be `None`. A Python object in a function definition will always point to the same object in memory (not a new empty dict each time)

Reply:
Oh no! Rookie error. Fixed in 10f7da0